### DISCUSSION TITLE
Home assistant Integration

### DISCUSSION THREAD

**Discussion by Coroebus** [ParadoxAlarmInterface/pai#357] [Q&A] [answered]
Hello,

I have successfully installed the HomeAssistant integration but I would like to confirm that it is indeed possible to arm or disarm a zone from HomeAssistant.

If so, who can tell me how to do it?

Thank you :)

**Comment by yozik04** [comment, marked as answer]
It is possible to arm/disarm partitions. I think everything is described in the wiki. You need to start there.

**Reply by RTAeroHASS** [reply]
Thanks

**Comment by RTAeroHASS** [comment]
Is it possible to bypass zone?

**Reply by yozik04** [reply]
https://github.com/ParadoxAlarmInterface/pai/wiki/Home-Assistant-integration#add-more-properties-to-zones-or-partitions
